graphics/pixelformat.h

```diff
@@ -139,12 +148,14 @@ struct PixelFormat {
 	byte rLoss, gLoss, bLoss, aLoss; /**< Precision loss of each color component. */
 	byte rShift, gShift, bShift, aShift; /**< Binary left shift of each color component in the pixel value. */
 
+	/** Define a pixel format ... @todo Doc required */
 	inline PixelFormat() {
 		bytesPerPixel =
 		rLoss = gLoss = bLoss = aLoss =
 		rShift = gShift = bShift = aShift = 0;
 	}
 
+	/** Define a pixel format ... @todo Is this RGB888? */
```

Comment:
It is not necessarily RGB888. It depends on the value you pass to the constructor.
RGB888 would be `BytesPerPixel` = 3, `RBits` = `GBits` = `BBits` = 8, `ABits` = 0, `RShift` = 16, `GShift` = 8, `BShift` = 0, `AShift = 0

So basically `BytesPerPixel` is the total number of bytes used to define a colour, `RBits`, `GBits`, `BBits`, and `ABits` are the number of bits for each component, and `RShift`, `GShift`, `BShift`, and `AShift` is the amount of bit shift for the component in the colour value.

Some other examples:
- RGBA8888 is `BytesPerPixel` = 4, `RBits` = `GBits` = `BBits` = `ABits` = 8, `RShift` = 24, `GShift` = 16, `BShift` = 8, `AShift = 0
- ABGR8888 is `BytesPerPixel` = 4, `RBits` = `GBits` = `BBits` = `ABits` = 8, `RShift` = 0, `GShift` = 8, `BShift` = 16, `AShift = 24
- RGB565 is `BytesPerPixel` = 2, `RBits` = 5, `GBits` = 6, `BBits` = 5, `ABits` = 0, `RShift` = 11, `GShift` = 5, `BShift` = 0, `AShift = 0